src/main/java/duke/command/RemoveCommand.java

```diff
@@ -0,0 +1,74 @@
+package duke.command;
+
+import duke.task.Task;
+import duke.util.TaskList;
+import duke.util.Ui;
+import duke.util.Storage;
+import duke.util.DukeException;
+
+/**
+ * The remove command allows user to indicate if they want to
+ * remove all tasks in the list or remove a specific task via
+ * a task number. Details are in the execute method.
+ */
+public class RemoveCommand implements Command {
+
+    private final int index;
+
+    private final boolean clear;
```

Comment:
Styling error. Boolean variables should be named to sound like booleans. Maybe you can use isClear instead of clear here.

Reply:
Good catch, thanks